### DISCUSSION TITLE
Please can you add Mercenaries 2 World in Flames BLES00323?

### DISCUSSION THREAD

**Discussion by Peoples00** [bucanero/ArtemisPS3#15] [Cheat requests]
can anyone add cheats for mercenaries 2 bles00323 version 1.03?

**Comment by bucanero** [comment]
You can find about how to make codes here: https://github.com/Dnawrkshp/NetCheatPS3

Here's a video tutorial for NetCheat: https://www.youtube.com/watch?v=2dxuoLTA_Lk

**Reply by Peoples00** [reply]
I can't create cheats with netcheat because I only use ps3hen and it also says that you shouldn't install CCAPI because of softbrick

**Reply by bucanero** [reply]
you can use it without CCAPI, there's a "fake ccapi" that works with PS3MAPI (available on HEN)
https://www.mediafire.com/file/znig9z0dsbg6x7k/Fake_CCAPI.rar/file

you can read about it here:
https://www.psx-place.com/threads/new-fake-ccapi-files-for-hen.24276/
